jmbase/jmbase/support.py

```diff
@@ -143,4 +210,68 @@ def lookup_appdata_folder(appname):
         data_folder = path.join(environ['APPDATA'], appname) + '\\'
     else:
         data_folder = path.expanduser(path.join("~", "." + appname + "/"))
-    return data_folder
\ No newline at end of file
+    return data_folder
+
+# helper functions for conversions of format between over-the-wire JM
+# and internal. See details in hexbin() docstring.
+
+def cv(x):
```

Comment:
convert() instead of cv()?

Reply:
It's only used internally in the module (see the next two functions), hence I was going for brevity. Perhaps I should add an underscore to hint that, as per Python convention? Indeed it'd be bad naming for anything exposed to users of the package.

Reply:
Changed to `_convert` in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/536/commits/0984273e60b52bf8e6d1afcada2b175f5afbb4d8